mmj2jar/PATutorial/Page407.mmp

```diff
@@ -4,15 +4,23 @@ $( <MM> <PROOF_ASST> THEOREM=reiteration  LOC_AFTER=
  Derive Feature. But first, note that as an experiment, step 5555 has
  been added; this will demonstrate deriving a formula!
 
- Now press Ctrl-U. Then, to tidy up, use Menu Item
+ Now press Ctrl-U. Notice that mmj2 figured out that since ax-mp
+ requires two hypotheses, but we only supplied on, it had to
```

Comment:
s/on/one

Reply:
Whups, you're absolutely right. I will fix that!